app/services/chart_builder.rb

```diff
@@ -1,35 +1,63 @@
 class ChartBuilder
-  def initialize(benchmark_runs)
+  
+  # @columns is JSON that looks like [{ name: "benchmark1", data: [1.1, 1.2] }]
```

Comment:
why `@columns` is JSON? The ruby class should operate with Ruby data structures. JSON is just a side effect of using serialization